### DISCUSSION TITLE
Game mode suggestion for city models.

### DISCUSSION THREAD

**Discussion by JavaGT** [SebLague/Geographical-Adventures#64] [Ideas]
I love the visuals of Paris in the video. I would actually be interested in smaller, address/landmark based delivery game to learn my way around the city I live in (Auckland, New Zealand).
Perhaps a city scale of version of the game where you drop off to local landmarks or suburbs could be a fun use of the city models.
I'm sure some twitter crowdsourcing, google scraping or hoodmaps.com could provide data on landmarks.

**Comment by SlashAcorn** [comment]
There could be something where people would use the method he used in the video and we could all contribute the places that we know about. Just and idea.
